docs/necronomicon.rst

```diff
@@ -0,0 +1,15 @@
+necronomicon
+============
+
+.. dfhack-tool::
+    :summary: Find books that contain the secrets of life and death
+    :tags: fort items
+
+Lists all books in the fortress that contain the secrets to life and death.
```

Comment:
It would be useful to include instructions for how to find the actual object in-game so the player can do something with the information.